etcdserver/v3demo_server.go

```diff
@@ -103,16 +105,23 @@ func (s *EtcdServer) Watchable() dstorage.Watchable {
 	return s.kv
 }
 
+const (
+	// noTxn is an invalid txn ID.
+	// To apply with independent Range, Put, Delete, you can pass noTxn
+	// to apply functions instead of a valid txn ID.
+	noTxn = -1
```

Comment:
noTxn could be a valid txn ID returned by TxnBegin, right? does this affect correctness?